crates/turborepo-ffi/messages.proto

```diff
@@ -89,3 +89,18 @@ message SubgraphResponse {
     string error = 2;
   }
 }
+
+message VerifySignatureRequest {
+  string hash = 1;
+  bytes artifact_body = 2;
+  string team_id = 3;
```

Comment:
`hash` and `team_id` being strings worry me since a Go string "holds arbitrary bytes" unlike Rust. 
I feel that we should either
 - Make this field bytes
 - Do verification on the Go side for `team_id` and error for a user if they try to use a non-utf8 team id, as mentioned previously I think we can verify `hash` by using a newtype for `PackageTask.Hash`

Reply:
I converted team_id to bytes. For hash, I feel like if we're now testing the accurate domain, i.e. random hashes, then it's okay to keep it as a string.